### ISSUE TITLE
Feature request: sort by last use, further view configuration options 

### ISSUE THREAD

**Issue by DrKGD** [Hajime-Suzuki/vuffers.nvim#39]
Hi there, I was looking **exactly** for a plugin like this, in fact, I tried to implement my very own version of it until I found this one! 

Consider this issue as a feature request / minor nitpick
1. I'd like to configure _some_ window options from the view configuration (e.g. I'd rather not have numbering enabled altogheter) , [there is already a neat table](https://github.com/Hajime-Suzuki/vuffers.nvim/blob/main/lua/vuffers/window.lua#L27) which could be extended via nvim builtin `vim.tbl_deep_extend`.

2. Sorting by filename is fine, but I'd like to ability to sort buffers by last use; on top of my head I don't think there is a builtin vim function for doing so, sorting by bufnr doesnt really cut it.

For clarity, let me explain what I mean, suppose three buffers are opened, thus the vuffers buffer would look like
```
A
B
C
```

Thus I either `:edit C`, issue the vuffers `open` action or any other third party plugins (fzf, telescope, nvim-tree... you get the idea), I'd like the list to be updated to
```
C
A
B
```

It is a shame I couldnt really find the plugin via search engines nor common plugin lists; consider adding it to awesome-neovim and/or dotfyle this-week-in-neovim, as it would really boost its deserved visiblity.
Thanks!

**Comment by Hajime-Suzuki** [comment]
Hi @DrKGD, thank you for your feedback and PR! I'll take a look at the PR in a few days.

Regarding sorting by last used, would you like to persist the last used time so that the order remains, after restarting neovim? I prefer to create as few external files as possible. But since I didn't think of the use case, I appreciate your input.

**Comment by DrKGD** [comment]
Not necessarily, I'd rather have it sorted by last use just for the current neovim instance, probably a buffer only variable which logs the timestamp on BufWinEnter would be enough for my use case (kinda like a reminder of which was my previous buffer)

In short I always desidered to navigate through previous buffers like you'd do with cybu.nvim, harpoon or vim markers, but I'd like to keep the comfort to always have a sorted list of previous buffers on screen to see, then quickly access them through custom bindings (`<C-x>1` would, for example, select the first previous buffer)

Let me also address that the whole PR, it only benefits my use case, feel free to discard any or all. of the changes, in particular the whole "open at a specific window" one

Thanks!

**Comment by Hajime-Suzuki** [comment]
Thanks! I will take a look at how to implement the sorting 👍

**Comment by DrKGD** [comment]
Just found out that each and every buffer already has a timestamp associated with it, you can check it out with `:ls t`,
you'd get the value for a single buffer via `vim.fn.getbufinfo(buffer)[1].lastused`, not sure whether or not there is a `vim.api` function instead

**Comment by Hajime-Suzuki** [comment]
It seems `lastused` counts only seconds, not milliseconds. If you switch buffers within a second, which is pretty normal, you might get the wrong order. I'm trying to use `vim.loop.now()` on my branch.

**Comment by Hajime-Suzuki** [comment]
I created a POC PR. Feel free to try it out.

**Comment by DrKGD** [comment]
Already testing it, thank you so much, will let you know :eyes:
